From: =?utf-8?b?6YeR5oWn54+g?= <2429894928@qq.com>
Date: Wed, 26 Aug 2026 13:47:01 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (24 of 24 strings)

Translation: openkylin-huanghe/ukui-menu-starter
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-menu-starter/ko_KP/
---
 widgets/ukui-menu-starter/i18n/org.ukui.menu.starter_ko_KP.ts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/widgets/ukui-menu-starter/i18n/org.ukui.menu.starter_ko_KP.ts b/widgets/ukui-menu-starter/i18n/org.ukui.menu.starter_ko_KP.ts
index 2bcb25f..eb6289e 100644
--- a/widgets/ukui-menu-starter/i18n/org.ukui.menu.starter_ko_KP.ts
+++ b/widgets/ukui-menu-starter/i18n/org.ukui.menu.starter_ko_KP.ts
@@ -9,11 +9,11 @@
     </message>
     <message>
         <source>Hibernate</source>
-        <translation>최대 절전</translation>
+        <translation>휴면</translation>
     </message>
     <message>
         <source>Suspend</source>
-        <translation>중지</translation>
+        <translation>수면</translation>
     </message>
     <message>
         <source>Restart</source>
@@ -56,7 +56,7 @@
     <name>main</name>
     <message>
         <source>Search</source>
-        <translation>전체 검색</translation>
+        <translation>검색</translation>
     </message>
     <message>
         <source>File Manager</source>
@@ -84,7 +84,7 @@
     </message>
     <message>
         <source>Power Manager</source>
-        <translation>전원 관리자</translation>
+        <translation>전원 관리</translation>
     </message>
     <message>
         <source>About This Computer</source>
